ec2api/api/common.py

```diff
@@ -343,6 +347,7 @@ def describe(self, context, ids=None, names=None, filter=None):
         self.names = set(names or [])
         self.items = self.get_db_items()
```

Comment:
Yes it returns list.
Snippet from ec2api/db/sqlalchemy/api.py

    return [_unpack_item_data(item)
            for item in (model_query(context, models.Item).
                         filter_by(project_id=context.project_id).
                         filter(models.Item.id.like('%s-%%' % kind)).
                         all())]